frontend/src/app/features/team-planner/team-planner/add-work-packages/add-existing-pane.component.ts

```diff
@@ -0,0 +1,146 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ElementRef,
+  ViewChild,
+} from '@angular/core';
+import { I18nService } from 'core-app/core/i18n/i18n.service';
+import { imagePath } from 'core-app/shared/helpers/images/path-helper';
+import {
+  BehaviorSubject,
+  of,
+} from 'rxjs';
+import { WorkPackageResource } from 'core-app/features/hal/resources/work-package-resource';
+import { ApiV3FilterBuilder } from 'core-app/shared/helpers/api-v3/api-v3-filter-builder';
+import {
+  catchError,
+  debounceTime,
+  map,
+} from 'rxjs/operators';
+import { ApiV3Service } from 'core-app/core/apiv3/api-v3.service';
+import { WorkPackageNotificationService } from 'core-app/features/work-packages/services/notifications/work-package-notification.service';
+import { CurrentProjectService } from 'core-app/core/current-project/current-project.service';
+import { UrlParamsHelperService } from 'core-app/features/work-packages/components/wp-query/url-params-helper';
+import { IsolatedQuerySpace } from 'core-app/features/work-packages/directives/query-space/isolated-query-space';
+import { UntilDestroyedMixin } from 'core-app/shared/helpers/angular/until-destroyed.mixin';
+import { CalendarDragDropService } from 'core-app/features/team-planner/team-planner/calendar-drag-drop.service';
+
+@Component({
+  selector: 'op-add-existing-pane',
+  templateUrl: './add-existing-pane.component.html',
+  styleUrls: ['./add-existing-pane.component.sass'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class AddExistingPaneComponent extends UntilDestroyedMixin {
```

Comment:
Perhaps add the `op-add-existing-pane` base BEM class via `@HostBinding` just to be correct in BEM application. Unfortunately angulars styling implementation doesn't allow us to select it, but I think it makes the rendered HTML more readable if it's clear what the base block is.